src/test/java/com/comphenix/protocol/events/PacketContainerTest.java

```diff
@@ -66,34 +89,27 @@
 import org.bukkit.potion.PotionEffect;
 import org.bukkit.potion.PotionEffectType;
 import org.bukkit.util.Vector;
-import org.junit.BeforeClass;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.powermock.core.classloader.annotations.PowerMockIgnore;
-
-import static com.comphenix.protocol.utility.TestUtils.*;
-import static org.junit.Assert.*;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
 
 // Ensure that the CraftItemFactory is mockable
-@RunWith(org.powermock.modules.junit4.PowerMockRunner.class)
-@PowerMockIgnore({ "org.apache.logging.log4j.core.config.xml.*", "javax.management.*" })
-//@PrepareForTest(CraftItemFactory.class)
 public class PacketContainerTest {
-	// Helper converters
-	private EquivalentConverter<WrappedDataWatcher> watchConvert = BukkitConverters.getDataWatcherConverter();
-	private EquivalentConverter<ItemStack> itemConvert = BukkitConverters.getItemStackConverter();
+
 	private static BaseComponent[] TEST_COMPONENT;
+	// Helper converters
+	private final EquivalentConverter<WrappedDataWatcher> watchConvert = BukkitConverters.getDataWatcherConverter();
+	private final EquivalentConverter<ItemStack> itemConvert = BukkitConverters.getItemStackConverter();
 
-	@BeforeClass
+	@BeforeAll
 	public static void initializeBukkit() {
-		BukkitInitialization.initializeItemMeta();
-		BukkitInitialization.initializePackage();
+		BukkitInitialization.initializeAll();
+		BukkitInitialization.initializeAll();

```

Comment:
Probably just a result of the find and replace, but this doesn't need to be called twice